requirements.txt

```diff
@@ -9,3 +9,5 @@ beautifulsoup4
 pyfiglet
 cowsay
 Pillow
+easyocr
+validators
```

Comment:
Is this `validators` really necessary? We already have a `is_url` function in the `utils.py`